### PR TITLE
fix: update Next.js version in create-next-app

### PR DESCRIPTION
**Pull request by dnhn**
### What?
The default template generated from `create-next-app` still says **Next.js 13**.

image

### Why?
Next.js 14 is released.

### CODE REVIEW

**Comment by ijjk** [comment]
Allow CI Workflow Run

- [ ] approve CI run for commit: 5e80a524921101f7ebb079db578d978a6bd5f642

Note: this should only be enabled once the PR is ready to go and can only be enabled by a maintainer

File: packages/create-next-app/templates/app-tw/js/app/page.js [hunk 1]
```diff
@@ -87,7 +87,7 @@ export default function Home() {
             </span>
           </h2>
           <p className={`m-0 max-w-[30ch] text-sm opacity-50`}>
-            Explore the Next.js 13 playground.
```
Comment (leerob) [inline packages/create-next-app/templates/app-tw/js/app/page.js:90]: Could we instead of "Explore starter templates for Next.js"

**Review by ijjk — APPROVED** [review]

**Comment by ijjk** [comment]
# Stats from current PR

<details open>
<summary><strong>Default Build</strong></summary>

<br/>

<details>
<summary><strong>General</strong></summary>

|  | vercel/next.js canary  | dnhn/next.js fix/create-next-app-version | Change |
| - | - | - | - |
| buildDuration | 10.2s | 11.8s | ⚠️ +1.6s |
| buildDurationCached | 7s | 6.4s | N/A |
| nodeModulesSize | 175 MB | 175 MB | ✓ |
| nextStartRea..uration (ms) | 404ms | 481ms | N/A |

</details>

<details>
<summary><strong>Client Bundles (main, webpack)</strong></summary>

|  | vercel/next.js canary  | dnhn/next.js fix/create-next-app-version | Change |
| - | - | - | - |
| 199-HASH.js gzip | 30 kB | 30 kB | N/A |
| 3f784ff6-HASH.js gzip | 53.2 kB | 53.2 kB | ✓ |
| 494.HASH.js gzip | 182 B | 182 B | ✓ |
| framework-HASH.js gzip | 45.5 kB | 45.5 kB | ✓ |
| main-app-HASH.js gzip | 254 B | 252 B | N/A |
| main-HASH.js gzip | 33.1 kB | 33.1 kB | N/A |
| webpack-HASH.js gzip | 1.75 kB | 1.75 kB | N/A |
| Overall change | 98.9 kB | 98.9 kB | ✓ |

</details>

<details>
<summary><strong>Legacy Client Bundles (polyfills)</strong></summary>

|  | vercel/next.js canary  | dnhn/next.js fix/create-next-app-version | Change |
| - | - | - | - |
| polyfills-HASH.js gzip | 31 kB | 31 kB | ✓ |
| Overall change | 31 kB | 31 kB | ✓ |

</details>

<details>
<summary><strong>Client Pages</strong></summary>

|  | vercel/next.js canary  | dnhn/next.js fix/create-next-app-version | Change |
| - | - | - | - |
| _app-HASH.js gzip | 205 B | 205 B | ✓ |
| _error-HASH.js gzip | 182 B | 181 B | N/A |
| amp-HASH.js gzip | 505 B | 507 B | N/A |
| css-HASH.js gzip | 322 B | 323 B | N/A |
| dynamic-HASH.js gzip | 2.59 kB | 2.59 kB | N/A |
| edge-ssr-HASH.js gzip | 258 B | 259 B | N/A |
| head-HASH.js gzip | 348 B | 347 B | N/A |
| hooks-HASH.js gzip | 369 B | 368 B | N/A |
| image-HASH.js gzip | 4.38 kB | 4.38 kB | N/A |
| index-HASH.js gzip | 256 B | 256 B | ✓ |
| link-HASH.js gzip | 2.67 kB | 2.67 kB | N/A |
| routerDirect..HASH.js gzip | 318 B | 318 B | ✓ |
| script-HASH.js gzip | 384 B | 383 B | N/A |
| withRouter-HASH.js gzip | 319 B | 320 B | N/A |
| 1afbb74e6ecf..834.css gzip | 106 B | 106 B | ✓ |
| Overall change | 885 B | 885 B | ✓ |

</details>

<details>
<summary><strong>Client Build Manifests</strong></summary>

|  | vercel/next.js canary  | dnhn/next.js fix/create-next-app-version | Change |
| - | - | - | - |
| _buildManifest.js gzip | 484 B | 484 B | ✓ |
| Overall change | 484 B | 484 B | ✓ |

</details>

<details>
<summary><strong>Rendered Page Sizes</strong></summary>

|  | vercel/next.js canary  | dnhn/next.js fix/create-next-app-version | Change |
| - | - | - | - |
| index.html gzip | 527 B | 527 B | ✓ |
| link.html gzip | 541 B | 543 B | N/A |
| withRouter.html gzip | 524 B | 522 B | N/A |
| Overall change | 527 B | 527 B | ✓ |

</details>

<details>
<summary><strong>Edge SSR bundle Size</strong></summary>

|  | vercel/next.js canary  | dnhn/next.js fix/create-next-app-version | Change |
| - | - | - | - |
| edge-ssr.js gzip | 96.1 kB | 96.1 kB | N/A |
| page.js gzip | 140 kB | 140 kB | N/A |
| Overall change | 0 B | 0 B | ✓ |

</details>

<details>
<summary><strong>Middleware size</strong></summary>

|  | vercel/next.js canary  | dnhn/next.js fix/create-next-app-version | Change |
| - | - | - | - |
| middleware-b..fest.js gzip | 625 B | 627 B | N/A |
| middleware-r..fest.js gzip | 148 B | 151 B | N/A |
| middleware.js gzip | 23 kB | 23 kB | ✓ |
| edge-runtime..pack.js gzip | 1.92 kB | 1.92 kB | ✓ |
| Overall change | 24.9 kB | 24.9 kB | ✓ |

</details>

</details>
Commit: 794ccf95e82282a65b84be3a460b7c29c435fd05

**Review by Paczjo — COMMENTED** [review]
dnhn:fix/create-next-app-version

**Review by Paczjo — COMMENTED** [review]
packages/create-next-app/templates/app-tw/js/app/page.js

**Pull request merged**